### ISSUE TITLE
Does not seem to work with recent .dat files from Prisma scanner running VE11E

### ISSUE THREAD

**Issue by mjherold** [pehses/twixtools#8]
First of all thank you for creating this tool!

When I try to read recent raw data files from a Prisma scanner running VE11E the read_twix function returns an empty list of scans. I can share the raw data file(s) which create this problems. It occurs with raw data from simple localizer scan and from other sequences too. Any ideas how to deal with this problem?

Thank you!

Best regards,

Michael

**Comment by pehses** [comment]
Hi Michael,
yes, it would be great if you could share a raw data file with me. I've never had any issues with VE11E data until now, so I'm curious what's going wrong in your case.
Best wishes,
Philipp

**Comment by mjherold** [comment]
Hi Philipp

The problem in BART’s twixread was fixed and with the current version that you clone from git it works fine.

But I did find a slight problem with twixtools: for some reason it is best to first save the raw data to the disk on the scanner computer and then copy it to a USB flash drive. Our Siemens applications scientist, Ravi Teja Seethamraju told me that Twix has a apparently slight bug where it “thinks” the file is finished being written to disk when in fact it is not finished, resulting in a corrupted file. And for some reason this only happens on one of our scanners, but writing first to local disk and then copying to flash usb drive reproducibly fixes the problem.

The problem with BART’s twixtread I posted about was independent of this issue with the Siemens Twix tool and the files were readable by the siemens-ismrmtools but not by twixread in BART until the issue was fixed.

I can still share raw data files if you like, but as I said the issue is fixed with twixread in BART and may have been an intermediate issue with a recent version of BART.

Best wishes,

Michael
